### PR TITLE
Week2: Exercise 2.1

### PR DESCRIPTION
**Pull request by AakashPandya**
image

### CODE REVIEW

File: Week-2/Exercise-2.1/index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Exercise 2.1</title>
+    <meta name="description" content="Pesto Plus program">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
```
Comment (RajeshSivanesan) [inline Week-2/Exercise-2.1/index.html:12]: @AakashPandya The page is responsive. But there should be some text to the right of the page. Based on the responsiveness, it has to wrap down the image. I think here its by default below the image

Can you please fix it

File: Week-2/Exercise-2.1/index.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise
+            on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit
+            amet..", comes from a line in section 1.10.32.</p>
+        </article>
+
+        <form action="" method="post" class="form">
+            <label for="title">Title</label><br>
+            <input type="radio" id="mr" name="title" value="Mr">
+            <label for="mr">Mr</label>
+            <input type="radio" id="miss" name="title" value="Miss">
```
Comment (RajeshSivanesan) [inline Week-2/Exercise-2.1/index.html]: @AakashPandya Form elements are not aligned properly. Looking very odd, can you provide some spacing and arrange it accordingly

**Review by RajeshSivanesan — COMMENTED** [review]
@AakashPandya Added comments, please check

File: Week-2/Exercise-2.1/index.html (hunk 2 above)
Comment (AakashPandya) [reply Week-2/Exercise-2.1/index.html]: Thanks @RajeshSivanesan , formatted html, css both files by using prettier code formatter extension. 
I will take care about this in future.

File: Week-2/Exercise-2.1/index.html (hunk 1 above)
Comment (AakashPandya) [reply Week-2/Exercise-2.1/index.html:12]: Updated code, so on a smaller device the right side content should stack below the image.
image

**Review by RajeshSivanesan — APPROVED** [review]

**Pull request merged**